soaktest/README.md

```diff
@@ -0,0 +1,39 @@
+# Information for soak test for go client
+The soak test contains 2 applications:
+soakclient.go for one application : Create Producer-1 to keep sending messages to the input-topic.
+Create Consumer-2 to receive messages from the output-topic.
+
+soakclient_transaction.go for the other application: Create Consumer-1 to receive messages from the input-topic.
+Create transactional producer Producer-2 to sending the received messages to the output-topic.
+The initTransaction API registers a transactional.id when creating the producer.
+The producer sends the messages to the output-topic.
+Commit transaction after sending 100 messages each time or 1 second.
+
+# Instruction on configuration and running
+
+# Build soaktest API
+    $ cd confluent-kafka-go/soaktest
+    $ go mod init soaktest
+    $ go mod tidy
+    $ go install soaktest
+   
+# Build and run soakclient.go
+    $ cd confluent-kafka-go/soaktest/soakclient
+    $ go build
+    $ ./soakclient -broker <..> -inputTopic <..> -outTopic <..> -groupId <..> -inputTopicPartitionsNum <..> -outTopicPartitionsNum <..> -replicationFactor <..> -ccloudAPIKey <ccloudAPIKey> -ccloudAPISecret <ccloudAPISecret>
+   
+# Build and run soakclient_transaction.go
+    $ cd confluent-kafka-go/soaktest/soakclient_transaction
+    $ go build
+    $ ./soakclient_transaction -broker <..> -inputTopic <..> -outTopic <..> -groupId <..> -transactionId <..> -ccloudAPIKey <ccloudAPIKey> -ccloudAPISecret <ccloudAPISecret>
+
+### The default values if doesn't input from the command lines
+    broker: "127.0.0.1:9092"
```

Comment:
I'm not very fond of default server addresses, even though the default broker is local it might be running for completely different purposes (some other test) and running these clients on that cluster without the user knowing might mess things up, so don't provide a default broker address.